core/src/main/java/org/testcontainers/utility/DockerImageName.java

```diff
@@ -63,14 +62,16 @@ public DockerImageName(String fullImageName) {
 
         if (remoteName.contains("@sha256:")) {
             repo = remoteName.split("@sha256:")[0];
-            versioning = new Sha256Versioning(remoteName.split("@sha256:")[1]);
+            versioning = new Versioning.Sha256Versioning(remoteName.split("@sha256:")[1]);
```

Comment:
tiny nit: if we import `Versioning.Sha256Versioning` and other `Versioning.*` classes, the changelog should be smaller :)

Reply:
Fixed

Reply:
@rnorth is it? 😅

Reply:
Oops, must have reverted it during moving the interface. Done again.